api/client/trust.go

```diff
@@ -37,7 +37,10 @@ import (
 	"github.com/docker/notary/tuf/data"
 )
 
-var untrusted bool
+var (
+	releasesRole = data.CanonicalTargetsRole + "/releases"
```

Comment:
minor (very minor, no need to update) nit, this should be `path.Join(data.CanonicalTargetsRole, "releases")`